package.json

```diff
@@ -8,7 +8,7 @@
     "test": "test"
   },
   "scripts": {
-    "test": "xo && ava test/**/*.test.js --require babel-register",
+    "test": "xo && ava test/**/*.test.js --require babel-register --verbose",
```

Comment:
If needed to do this, do in a commit only for this, explaining why.

Reply:
you are right, sry.